src/collections/_documentation/workflow/notifications/alerts.md

```diff
@@ -3,22 +3,129 @@ title: 'Alerts'
+    - interval: one minute, one hour, one day, one week, or 30 days
+- An event's {attribute} value {match} {value}
+    - attribute: message, platform, environment, type, exception.type, exception.value, user.id, user.email, user.username, user.ip_address, http.method, http.url, stacktrace.code, stacktrace.module, or stacktrace.filename
+    - match: equals, does not equal, starts with, ends with, contains, does not contain, is set, or is not set
+    - value: any attribute's value
+- An event's level is {match} {level}
+    - match: equal to, less than or equal to, greater than or equal to
+    - level: fatal, error, warning, info, debug, or sample
 
-An example use-case might be that you want a specific kind of error to send via Slack, but you want everything to send via Mail.
\ No newline at end of file
+## Actions
+
+Actions are located in **Project Settings > Alerts > Rules > "New Alert Rules" or "Edit Rule"**.
+
+[{% asset notifications/new_alert_rule.png alt="Buttons to create a New Alert Rule or to Edit an existing rule." %}]({% asset notifications/new_alert_rule.png @path %})
+
+An example use-case might be that you want a specific kind of error to send via Slack, but you want everything to send via Mail.
+
+### Action Types
+
+Currently, there are three types of actions you can take:
+
+- Send a notification (for all legacy integrations)
+    - Per-Project Integrations are considered to be legacy integrations
+- Send a notification via {service}
+    - service is one of mail and any configured per-project integration
+- Send a notification to the {workspace} Slack workspace to {channel} and show tags {tags} in notification
+    - to clarify this relates to the global Slack integration, *not* the per-project legacy integration
+
+[{% asset notifications/alert_actions.png alt="Dropdown of notification actions." %}]({% asset notifications/alert_actions.png @path %})
+
+An example use-case might be that you want a specific kind of error to send via Slack, but you want everything to send via Mail.
+
+### Frequency
+
+- Perform these actions at most once every {interval} for an issue
+    - interval: 5 minutes, 10 minutes, 30 minutes, 60 minutes, 3 hours, 12 hours, 24 hours, one week, or 30 days
+    - Frequency configuration can get complicated. For example, if an issue changes state from resolved to unresolved, and then this issue is resolved, unresolved, and resolved again within 5 minutes, but your frequency threshold is 30 minutes, you'll only get one notification
+
+[{% asset notifications/alert_frequency.png alt="Dropdown of minute and hour options for frequency of notifications." %}]({% asset notifications/alert_frequency.png @path %})
+
+## Environment
+
+- All Environments
+- any one of your defined (and not hidden) environments
+
+[{% asset notifications/alert_environment.png alt="Dropdown of environments. Currently, only All Environments is an option." %}]({% asset notifications/alert_environment.png @path %})
```

Comment:
it might be nice if the screenshot here shows some other envs like "All Environments, production, staging". I can help you with that if you want to do so

Reply:
@ceorourke Yeah, will you send me a screenshot. :D Thank you!